### DISCUSSION TITLE
Yale Keyfree connected lock not compatible

### DISCUSSION THREAD

**Discussion by DirkFace** [zwave-js/zwave-js#2705] [General] [answered]
Hi, I've just migrated from smartthings over to HA & managed to move over all my devices that were on zigee or zwave - apart from my front door lock which is ... https://help.yale.co.uk/wp-content/uploads/2019/06/Keyfree-manual-2014-2.pdf

My back door lock is a connexis L1 & that works brilliantly.

I have minimal coding experience, so am reaching out here for some assistance if possible?
The zwave module is found when adding the device & is alive / beaming, but returns a Device ID NaN-NaN-NaN (0xXXXX-0xXXXX-0xXXXX), unknown manufacturer, etc.

As I mention, it worked fine thorugh zwave / smartthings & I was using the Rboy device handler.

**Comment by AlCalzone** [comment, marked as answer]
Please open a bug report and fill out the template there, especially including the requested logs.

**Reply by DirkFace** [reply]
Thanks for the quick reply. I'll try my best to to send over the correct info (newbie).
